src/components/Select/ComboboxMulti/__tests__/ComboboxMulti.spec.js

```diff
@@ -81,34 +81,46 @@ describe('ComboboxMulti', () => {
             expect(inputNode).toBe(document.activeElement);
             expect(wrapper.find('li')).not.toHaveLength(0);
         });
-        it('should blur on pressing Escape button', () => {
+        it('should blur on pressing Escape button', (done) => {
             const blurSpy = jest.spyOn(inputNode, 'blur');
 
             setFocusOnInput();
 
             expect(inputNode).toBe(document.activeElement);
 
-            wrapper.find('input').simulate('keyDown', { key: 'Escape' });
+            wrapper.find('input').simulate('blur').simulate('keyDown', { key: 'Escape' });
 
-            expect(inputNode).not.toBe(document.activeElement);
-            expect(blurSpy).toHaveBeenCalled();
-            expect(mockOnBlur).toHaveBeenCalled();
+            setTimeout(() => {
+                wrapper.update();
+                expect(inputNode).not.toBe(document.activeElement);
+                expect(blurSpy).toHaveBeenCalled();
+                expect(mockOnBlur).toHaveBeenCalled();
+                done();
+            }, 0);
```

Comment:
0 as a parameter can be omitted since it's a default value for the second argument

Reply:
done!